resources/microvm-kernel-x86_64.config

```diff
@@ -543,6 +541,8 @@ CONFIG_MEMORY_HOTPLUG_SPARSE=y
 CONFIG_MEMORY_HOTREMOVE=y
 CONFIG_SPLIT_PTLOCK_CPUS=4
 CONFIG_ARCH_ENABLE_SPLIT_PMD_PTLOCK=y
+CONFIG_MEMORY_BALLOON=y
+CONFIG_BALLOON_COMPACTION=y
```

Comment:
This is useful when guest is using hugepages as it helps improving memory defragmentation but at the cost of CPU overhead of an extra thread doing background work: https://elixir.bootlin.com/linux/latest/source/drivers/virtio/virtio_balloon.c#L753. 

Are we sure we want to have this by default ? I would think about a separate kernel configuration tuned for a specific usecase that requires guest huge page support.

Reply:
@georgepisaltu what is your thinking about this ?

Reply:
I already talked to @georgepisaltu , these options are automatically selected by VIRTIO_BALLON. George removed them manually when updating the config with VIRTIO_BALLON.